ethereum/env/.env.worldchain.testnet

```diff
@@ -0,0 +1,21 @@
+# Worldchain testnet env
+# Rename to .env
+
+# Common config for forge deployment
+RPC_URL=https://worldchain-sepolia.g.alchemy.com/public
+# FORGE_ARGS="--legacy"
+
+# Wormhole Core Migrations
+INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
+INIT_CHAIN_ID=45
+INIT_GOV_CHAIN_ID=0x1
+INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+INIT_EVM_CHAIN_ID=4801
+
+# Bridge Migrations
+BRIDGE_INIT_CHAIN_ID=45
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+BRIDGE_INIT_FINALITY=1
+# I'm assuming this is right. . .
+BRIDGE_INIT_WETH=0x211Db8FbDc34982654e39b1B3a8Ca3EF5c7826EA
```

Comment:
as an OP Stack chain, shouldn't this be `0x4200000000000000000000000000000000000006`?